modules/ROOT/pages/veevavault/veevavault-connector-studio.adoc

```diff
@@ -0,0 +1,177 @@
+
+WARN 2019-06-06 10:39:45,098 [[MuleRuntime].io.27: [veevavault-connector-demo].get-documents-operationFlow.BLOCKING @2dd2ffaf] [event: 99716950-8818-11e9-90e2-d61515f42856] org.mule.extension.veevavault.internal.operations.VeevaVaultOperations: Properties or Fields is not available for api [/api/v18.2/metadata/objects/documents/types/site_management__c] , will use input list [[name__v, type__v, subtype__v, classification__v, lifecycle__v, status__v, study__v, product__v, id]] if configured to build VQL query
+
+INFO 2019-06-06 10:39:45,098 [[MuleRuntime].io.27: [veevavault-connector-demo].get-documents-operationFlow.BLOCKING @2dd2ffaf] [event: 99716950-8818-11e9-90e2-d61515f42856] org.mule.extension.veevavault.internal.pagination.GetDocumentsPagingProvider: Query Build :: SELECT name__v, type__v, subtype__v, classification__v, lifecycle__v, status__v, study__v, product__v, id FROM documents WHERE name__v= 'F22611234--6764'
+
+INFO 2019-06-06 10:39:46,024 [[MuleRuntime].io.27: [veevavault-connector-demo].get-documents-operationFlow.BLOCKING @2dd2ffaf] [event: 99716950-8818-11e9-90e2-d61515f42856] org.mule.extension.veevavault.internal.service.HttpRequestService: Response received for sync http request :: 'https://[vault-domain]/api/v18.2/query?q=SELECT+name__v%2C+type__v%2C+subtype__v%2C+classification__v%2C+lifecycle__v%2C+status__v%2C+study__v%2C+product__v%2C+id+FROM+documents+WHERE+name__v%3D+%27F22611234--6764%27++LIMIT+1000+OFFSET+0' in 925 milliseconds.
+----
++
+[%header%autowidth.spread]
+|===
+|*Field* |*Description*
+|*API Limit Threshold* |Enter the VeevaVault API threshold limit. When this threshold is reached or exceeded, the connector waits the specified time. The default is `200`.
+|*Wait Time* |Enter the amount of time the connector should wait if the API limit threshold is reached or exceeded. The default is `5`.
+|*Time Unit* |Select the time unit for the wait time. The default is `MINUTES`.
+|===
++
+*Daily API Limit Exceed Profile:*
++
+The daily API limit exceed profile configuration (*Connector Configuration > Advanced Tab*) to use if the daily API limit of the Veeva Vault API reaches or exceeds the configured threshold. The daily limit is the total number of API calls within any 24-hour window on a daily rolling basis.
++
+If the *Daily API Limit Exceed Profile* is enabled (default), then the connector evaluates the Veeva Vault API's response header (*X-VaultAPI-DailyLimitRemaining*) for the specified value of the threshold, and throws an exception indicating the daily API limit is exceeded. The exception continues to be thrown for each and every job or request created until the daily limit is reset or rolled over to the next day.
```

Comment:
ditto prev. suggestion for rewording based on "default"

Reply:
"indicating" > "indicating that"

Reply:
"each and every" is idiomatic, although not incorrect.
It is a more forceful way of saying "each," and that is what I would use instead, for Global English.